ADAL/src/telemetry/ADTelemetryDefaultEvent.h

```diff
@@ -30,14 +30,15 @@
     NSInteger _defaultPropertyCount;
 }
 
+- (NSDictionary*)getProperties;
```

Comment:
In the original code, this was a private method which returns a private dictionary. The change is to make the method public. I will keep it this way to avoid large changes.